tests/integration/events.js

```diff
@@ -3,16 +3,23 @@ var test = require('tape')
 
 var MockStore = require('../utils/mock-store')
 var TaskQueue = require('../../index')
+var memdown = require('memdown')
+
+var options = {
+  remoteBaseUrl: '/api/users',
+  remote: 'test'
+}
 
 test('Events: typedQueue.on', function (t) {
   t.plan(3)
 
-  simple.mock(TaskQueue.internals, 'CustomStore').callFn(function (id) {
-    return new MockStore(id)
+  simple.mock(TaskQueue.internals, 'Store').callFn(function (options) {
+    options.db = memdown
+    return MockStore(options)
   })
 
-  var taskQueue = new TaskQueue('id123')
-  var fooQueue = taskQueue('foo')
+  var taskQueue = new TaskQueue('id123', options)
+  var fooQueue = taskQueue('foo', options)
```

Comment:
awesome! Ping me when it's ready for review. So cool to see all the progress in Hoodie land :v: